### PR TITLE
Bumps telemetry expiration dates to 2020-11-01.

### PR DESCRIPTION
**Pull request by daoshengmu**
Extend the expiration dates from 2020-05-01 to 2020-11-01.

### CODE REVIEW

**Comment by daoshengmu** [comment]
# Request for data collection review form
 **All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.**
  
 1. **What questions will you answer with this data?**

https://github.com/MozillaReality/FirefoxReality/pull/2241#issuecomment-555285627
- How many domains are visited in a session?
- How many URLs are visited in a session?
- How many times do users insert URL, search query, and voice query in the URL bar in a session?
- How many searches are queried in a specific search engine in a session?

https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
Tabs Telemetry:

- How many users ever opened a tab
- Average number of opened tabs/session (counting only users that have opened at least one tab)
- Average number of activated tabs/session (counting only users that have activated at least one tab)
- Opened tabs per session
- Number of tabs explicitly opened per session. (tabs can also be opened automatically after restoring a session, we don't want to count those here)
- Percentage of tabs opened via long press vs the tabs panel

FxA Telemetry:

- Number of users who click on the Sign-In button
- Number of users who complete the Sign-In flow successfully
- Number of users who Sign-Out
- Number of users who turn syncing for both Bookmarks and History
- Number of users who send a Tab from FxR
- Number of users who receive a tab from another device
- Number of users who have ever sent a tab
- Average number of tabs sent per week per user (counting only users that have opened at least one tab)
- Number of users who have ever received a tab
- Average number of tabs received per week per user (counting only users that have opened at least one tab)
- Number of tabs received for device type (Mobile/Desktop/Tablet/TV/VR)

https://github.com/MozillaReality/FirefoxReality/pull/2348#issuecomment-558835389
 - Counting how many general windows are opened in a session.

 2. **Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements?**

https://github.com/MozillaReality/FirefoxReality/pull/2241#issuecomment-555285627
- To measure how users insert URL into the URL bar, by inserting? query? or voice input? We could provide greater experience for searches.
- We wanna understand which search engine is the most users would like to use.
- We also are interested in analyzing users' behavior during a session about how often they use search engine and the search bar.

https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
- For Tabs, we want to better understand how people use and interact with tabs in a VR environment to help use improve the tabs user experience in future versions.
- For FxA, we want to know how many users use the feature, help discover possible issues with the sign in flow and the overall FxA integration experience and
better understand how people use FxA inside FxR to offer a better user experience in future versions.

https://github.com/MozillaReality/FirefoxReality/pull/2348#issuecomment-558835389
- We also are interested in analyzing users' behavior during a session about how they use FxR multi-window feature.
  
 3. **What alternative methods did you consider to answer these questions? Why were they not sufficient?**
 
 * N/A
 
 4. **Can current instrumentation answer these questions?**
 
 * We do, and we wanna extend the current telemetry expiration dates.
 
 5. **List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox [data collection](https://wiki.mozilla.org/Firefox/Data_Collection) [categories](https://wiki.mozilla.org/Firefox/Data_Collection#Data_Collection_Categories) on the found on the Mozilla wiki.**
 
 * All data is Category 2.
 
 6. **How long will this data be collected?**
 
Until re-evaluation 11/01/2020
 
 7. **What populations will you measure?**

 * All release, beta, and nightly users with telemetry enabled.
 
 8. **Please provide a general description of how you will analyze this data.**
 
 * Glean
 
 9. **Where do you intend to share the results of your analysis?**
 
 * Only on Glean and with Firefox Reality team.

**Comment by daoshengmu** [comment]
> # Request for data collection review form
Per https://github.com/MozillaReality/FirefoxReality/pull/3199#issuecomment-616716151

@harraton Please help data review.

The metrics document is here, https://github.com/MozillaReality/FirefoxReality/blob/bumpGleanDates/docs/metrics.md, and it will be shown in docs/metrics.md once we merge it.

The wiki page is here, https://github.com/MozillaReality/FirefoxReality/wiki/Telemetry#new-telemetry-system---glean

**Comment by daoshengmu** [comment]
@Dexterp37 Please help review as well.

File: app/metrics.yaml [hunk 1]
```diff
@@ -21,10 +21,11 @@ distribution:
       - https://github.com/MozillaReality/FirefoxReality/issues/1420
     data_reviews:
       - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+      - https://github.com/MozillaReality/FirefoxReality/pull/3199
```
Comment (Dexterp37) [inline app/metrics.yaml:24]: This, and all the others, need to point to the **data review response**.

**Comment by harraton** [comment]
> > # Request for Data Collection Renewal
> 
> Per [#3199 (comment)](https://github.com/MozillaReality/FirefoxReality/pull/3199#issuecomment-616716151)
> 
> @harraton Please help data review.
> 
> The metrics document is here, https://github.com/MozillaReality/FirefoxReality/blob/bumpGleanDates/docs/metrics.md, and it will be shown in docs/metrics.md once we merge it.
> 
> The wiki page is here, https://github.com/MozillaReality/FirefoxReality/wiki/Telemetry#new-telemetry-system---glean

Renewal:

    Is the provided Data Collection Review complete, correct, and data-review+ by a Data Steward?

YES

    Is the data collection covered by the existing Firefox Privacy Notice?

YES (Firefox Reality and Firefox Privacy Statements).

Renewal approved.

**Pull request merged**